android_jni/avifandroidjni/src/main/jni/libavif_jni.cc

```diff
@@ -146,6 +146,14 @@ avifResult AvifImageToBitmap(JNIEnv* const env,
     }
     image = cropped_image.get();
   }
+  if (isScale) {
+    res = avifImageScale(image, bitmap_info.width, bitmap_info.height, &decoder->decoder->diag);
```

Comment:
Vignesh: I suggest we remove the `diag` parameter from the `avifImageScale()` function first.

Until that is done, this pull request should declare a local `avifDiagnostics diag` variable and pass `&diag` to `avifImageScale()`.

Reply:
I have modified this problem